src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -1,7 +1,6 @@
-<br>
-<ng-container>
-  <div class="card bg-primary">
-    <div class="card-body fill-plain">
+<div class="container">
```

Comment:
The proportion is misaligned now:
![image](https://user-images.githubusercontent.com/25320782/52902989-6cc47700-3253-11e9-8361-c01a79166510.png)

I suspect the `container` class is adding extra padding which we do not need. In this case, you can declare your own class to do the following you proposed:
`For aesthetic purposes, I also added some whitespace in between the active, archived, and deleted courses section`.

Reply:
My bad, I didn't realize that the name `container` has already been used somewhere else. Will change the name!